service/controller/clusterapi/v31/resource/tccp/template/template_main_record_sets.go

```diff
@@ -67,25 +67,6 @@ const TemplateMainRecordSets = `
       Name: '{{ $v.EtcdDomain }}.'
       HostedZoneId: !Ref 'HostedZone'
       Type: A
-  IngressRecordSet:
-    Type: AWS::Route53::RecordSet
-    Properties:
-      AliasTarget:
-        DNSName: !GetAtt IngressLoadBalancer.DNSName
-        HostedZoneId: !GetAtt IngressLoadBalancer.CanonicalHostedZoneNameID
-        EvaluateTargetHealth: false
-      Name: 'ingress.{{ $v.ClusterID }}.k8s.{{ $v.BaseDomain }}.'
-      HostedZoneId: !Ref 'HostedZone'
-      Type: A
-  IngressWildcardRecordSet:
```

Comment:
As discussed here https://gigantic.slack.com/archives/C3S482Z98/p1571996691134200, this wildcard record should be kept.

Reply:
Fixed, PTAL